src/main/java/org/folio/rest/impl/PatronServicesResourceImpl.java

```diff
@@ -152,9 +154,15 @@ public void postPatronAccountByIdItemByItemIdHold(String itemId, String id,
         .put(JSON_FIELD_ITEM_ID, itemId)
         .put("requesterId", id)
```

Comment:
Should these fields' names (requesterId, requestType, requestDate, fulfillmentPreference) be made constants as well?

Reply:
They could be. Sonar was complains when there is more than X (3?) references to the same string. I didn't want to change too much. I think we can sort that out when we work on title based holds.